sources/accessControlLib.move

```diff
@@ -162,6 +168,13 @@ module peeranha::accessControlLib {
         revokeRole_(userRolesCollection, PROTOCOL_ADMIN_ROLE, userId);
     }
 
+    public(friend) fun setRoleAdminPublic(userRolesCollection: &mut UserRolesCollection, adminId: ID, role: vector<u8>, adminRole: vector<u8>) {
```

Comment:
Do we have tests for this function?
Please add. It is very critical part of code.